api/src/main/java/org/openmrs/module/fhir2/api/dao/impl/BaseDaoImpl.java

```diff
@@ -812,6 +823,12 @@ protected void handleResourceCode(Criteria criteria, TokenOrListParam code, @Not
 		}).ifPresent(criteria::add);
 	}
 	
+	protected void handleResourceCode(Criteria criteria, TokenOrListParam code, @NotNull String conceptAlias,
+	        @NotNull String conceptMapAlias, @NotNull String conceptReferenceTermAlias) {
+		
+		handleCode(criteria, code, conceptAlias, conceptMapAlias, conceptReferenceTermAlias);
+	}
```

Comment:
Let's get rid of this altogether, even though that means updating more files.

Reply:
@ibacher I'll do that but it'll take some time